frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segment_statistics_modal.tsx

```diff
@@ -56,38 +56,33 @@ type SegmentInfo = {
 };
 
 const exportStatisticsToCSV = (
-  segmentInformation: Array<SegmentInfo> | string,
+  segmentInformation: Array<SegmentInfo> | null,
   tracingId: string,
   groupIdToExport: number,
   hasAdditionalCoords: boolean,
 ) => {
-  if (segmentInformation.length === 0 || segmentInformation instanceof String) {
-    return;
-  }
   const segmentStatisticsAsString = (segmentInformation as Array<SegmentInfo>)
```

Comment:
this cast in this code version is now wrong (because `segmentInformation` might be null according to the functions typing) and should be removed. if `null` is not a valid parameter type anymore, it shouldn't be necessary, anymore.